### ISSUE TITLE
Feat: Disallow commenting a comment (backend)

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#3527]
## Feature description and goals

The backend should disallow sending a comment on a comment. This is not handled in Tracim.

**Comment by lebouquetin** [comment]
The backend should not disallow this: the content structure is generic and it is desired. This is to the frontend to manage correctly behavior. 

The reason for this is: improving the feature would be only frontend development instead of requiring both frontend and backend upgrades

**Comment by lebouquetin** [comment]
this is the same contents: the REST Api allows to define a content as a child of another content. (this will allow, for example, to manage files in history thread, etc)

**Comment by inkhey** [comment]
@lebouquetin Idea here is having some kind of assert here to disallow case the frontend doesn't support now, we can decide to not disallow this but : 
- We need to be very careful as this is difficult to debug unwanted use of "hidden" feature in frontend and it may create weird stuff in database.
- in this specific case, we do not support properly all the complexity of a comment tree in  last activity view (see screenshot: https://github.com/tracim/tracim/issues/3525).

**Comment by raphj** [comment]
@lebouquetin yes, but what happens if another tracim client adds a comment to a comment?

Bad things will happen with the frontend. Shouldn't we think about this feature carefully before allowing it? This effectively allows discussion trees in the comments, while we are currently using a flat list.

**Comment by lebouquetin** [comment]
@raphj this has always been there in the backend. This is not a bug but a misuse.

What is the explanation for the frontend to get unecessary/not understandable comments?

**Comment by raphj** [comment]
After discussion, we settled on keeping the possibility to comment a comment from the backend. It's okay to have features in the backend that are not exploited in the frontend.

I still fear that comments added by another client of the backend to a comment would be hidden to the frontend and people might not notice the issue (invisible problem), so this is something that we should keep in mind as soon as we have an alternative Tracim client.
